pkg/actors/actors_test.go

```diff
@@ -112,7 +112,7 @@ func deactivateActorWithDuration(testActorsRuntime *actorsRuntime, actorKey stri
 	testActorsRuntime.startDeactivationTicker(scanInterval, actorIdleTimeout)
 }
 
-func createReminder(actorID, actorType, name, period, dueTime, data string) CreateReminderRequest {
+func initializeReminderData(actorID, actorType, name, period, dueTime, data string) CreateReminderRequest {
```

Comment:
Can you change the method name to `createReminderData`? It's generally advisable in Go to use `initialize` as little as possible as a prefix to a function name